config.json

```diff
@@ -5,7 +5,13 @@
   "active": false,
   "test_pattern": "TODO",
   "exercises": [
-
+    {
+      "slug": "leap",
+      "difficulty": 1,
+      "topics": [
+        "generators"
```

Comment:
@SaschaMann yes, I agree, just used this from your pull request.
Are these three okay: Control-flow (conditionals), Integers, Mathematics?